deploy/staging/chainlog-logger.yaml

```diff
@@ -0,0 +1,64 @@
+replicaCount: 1
+service:
+  name: chainlog-logger
+  port: 8080
+  type: ClusterIP
+  containerPort: 8080
+  tls:
+    enabled: false
+image:
+  repository: 068992353948.dkr.ecr.eu-central-1.amazonaws.com/chainlog-logger-staging
```

Comment:
Can we pass variables into this file? Not great having account number hardcoded here

Reply:
@eskp done